src/components/Header/Header.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Header from './Header';
+export default {
+  title: 'Components/Header',
+  component: Header,
+} as ComponentMeta<typeof Header>;
+
+const Template: ComponentStory<typeof Header> = (args) => <Header {...args} />;
+
+export const Editor = Template.bind({});
+Editor.args = {
+  logo: 'https://editor.factiverse.no/79b028bc000a5e47c9a7.png',
```

Comment:
This doesn't get displayed correctly:
<img width="955" alt="Screenshot 2022-05-23 at 15 14 57" src="https://user-images.githubusercontent.com/44401648/169827887-2f50d47c-b063-4c86-a595-619c201e5a3c.png">
I think we should instead have a "img" folder like in ai-editor and have the logo in there instead of relying on the link.